packages/core/src/login/oidc/IClientRegistrar.ts

```diff
@@ -77,8 +77,7 @@ function determineClientType(
     return "static";
   }
   if (
-    issuerConfig.solidOidcSupported ===
-      "https://solidproject.org/TR/solid-oidc" &&
+    issuerConfig.scopesSupported.includes("webid") &&
```

Comment:
@NSeydoux this'll break NSS, as it doesn't advertise scopes supported, for example: https://solidcommunity.net/.well-known/openid-configuration